backend/layer_build/python/dateutil/tz/win.py

```
@@ -0,0 +1,370 @@
+
+        :return:
+            Returns the localized timezone string from tzres.dll if the string
+            is of the form `@tzres.dll,-offset`, else returns the input string.
+        """
+        if not tzname_str.startswith('@'):
+            return tzname_str
+
+        name_splt = tzname_str.split(',-')
+        try:
+            offset = int(name_splt[1])
+        except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)